### ISSUE TITLE
Add B1 shimming related functions to +imutils package

### ISSUE THREAD

**Issue by gaspardcereza** [shimming-toolbox/shimming-toolbox-matlab#147]
## Context

We want the shimming toolbox to handle B0 and B1 shimming. So far, only B0 shimming was performed but I'm currently working on finding a way to include RF shimming features as we refactor our code. 

## Objectives

Add functions to perform B1 mapping and shimming to the `+imutils` package (under `+imutils/+b1`). These functions will have to be documented following the convention that will be defined in #133. 
Unit tests and testing data must also be provided.

**Comment by jcohenadad** [comment]
i would suggest to use lowercase, i.e. `+imutils/+b1`, as also done for the `+b0` package https://github.com/shimming-toolbox/shimming-toolbox/pull/125/files

**Comment by gaspardcereza** [comment]
> i would suggest to use lowercase, i.e. `+imutils/+b1`, as also done for the `+b0` package https://github.com/shimming-toolbox/shimming-toolbox/pull/125/files

I thought you once said that b0 could be misguiding because it could also refer to another term (I can't find the comment though so maybe I just dreamed about it).

**Comment by jcohenadad** [comment]
> > i would suggest to use lowercase, i.e. `+imutils/+b1`, as also done for the `+b0` package https://github.com/shimming-toolbox/shimming-toolbox/pull/125/files
> 
> I thought you once said that b0 could be misguiding because it could also refer to another term (I can't find the comment though so maybe I just dreamed about it).

indeed, in the MRI community "b0" often refers to the b-value =0 in DWI experiments. However, in this context, there should not be too much confusion (it is clear we talk about static B0 field), and i find the pros more appealing:
- sticking to snake-case convention for functions/packages
- easier to type on the keyboard (no need for SHIFT)
